index.js

```diff
@@ -225,7 +222,7 @@ export function getHSV(color) {
     h /= 6;
   }
 
-  return [h, s, v, color[3]];
+  return [h, s, v, a];
```

Comment:
same as in getRGBBytes this function is not getHSVA

Reply:
Same here, in v1 it actually sets `color[3]` so if color is just [r, g, b], you get `[r, g, b, undefined]`.